app/src/main/res/values/attrs.xml

```diff
@@ -20,6 +20,7 @@
     <item name="android:background">@drawable/card_background</item>
     <item name="android:foreground">@drawable/card_touch_feedback</item>
     <item name="android:elevation">@dimen/card_elevation</item>
+    <item name="android:stateListAnimator">@anim/card_state_list_anim</item>
```

Comment:
Shouldn't we keep the style into a `styles.xml` files, instead of the `attrs.xml`?